base/loader/src/gtfs/gtfs_parser.cc

```diff
@@ -161,16 +160,16 @@ void gtfs_parser::parse(fs::path const& root, FlatBufferBuilder& fbb) {
   };
 
   motis::logging::scoped_timer export_timer{"export"};
-  auto const p = projection{0.4, 0.8};
   std::clog << '\0' << 'S' << "Export schedule.raw" << '\0';
```

Comment:
Macht es Sinn, das auch noch in den Call zu packen:

`motis::logging::import_step("Export schedule.raw", 40, 80, trips.size());`

Reply:
ja, macht sinn